Multisig/UI/App/UIKit/CollectiblesViewController/CollectiblesHeaderView.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  CollectiblesHeaderView.swift
+//  Multisig
+//
+//  Created by Andrey Scherbovich on 02.11.20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import UIKit
+
+class CollectiblesHeaderView: UITableViewHeaderFooterView {
+    @IBOutlet weak var headerImageView: UIImageView!
+    @IBOutlet weak var headerNameLabel: UILabel!
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        backgroundView = UIView(frame: self.bounds)
+        backgroundView!.backgroundColor = .gnoSystemWhite
+        headerNameLabel.setStyle(.headline)
+    }
+
+    func configure(collectibleSection: CollectibleListSection) {
```

Comment:
Here, too, the view depends on the model, and hence, is less reusable than it could.

I'd propose to have 2 methods:

```
setName(_ value: String)
setImage(with URL: URL, placeholder: UIImage)
```

Reply:
done